### PR TITLE
Widget->XObject

### PR DESCRIPTION
**Pull request by jdfreder**

### CODE REVIEW

File: examples/Widget Basics.ipynb [hunk 1]
```diff
@@ -196,7 +196,7 @@
    "source": [
     "Widgets are represented in the back-end by a single object.  Each time a widget is displayed, a new representation of that same object is created in the front-end.  These representations are called views.\n",
     "\n",
-    "![Kernel & front-end diagram](images/WidgetModelView.png)"
```
Comment (jdfreder) [inline examples/Widget Basics.ipynb]: wrong

Comment (ellisonbg) [inline examples/Widget Basics.ipynb]: Can you summarize the main points of the PR in a comment?

On Thu, Apr 7, 2016 at 10:55 AM, Jonathan Frederic <notifications@github.com
> wrote:

> In examples/Widget Basics.ipynb
> <https://github.com/ipython/ipywidgets/pull/464#discussion_r58915868>:
>
> > @@ -196,7 +196,7 @@
> >     "source": [
> >      "Widgets are represented in the back-end by a single object.  Each time a widget is displayed, a new representation of that same object is created in the front-end.  These representations are called views.\n",
> >      "\n",
> > -    "Kernel & front-end diagram"
>
> wrong
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipywidgets/pull/464/files/01034f4d9f5da7684f2f7bd4da099cc6b6ad5234#r58915868>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

Comment (jdfreder) [inline examples/Widget Basics.ipynb]: It's just an API rename, spawned from a discussion between @SylvainCorlay and I.  The PR is not actually meant to merged.

**Comment by ellisonbg** [comment]
Ah, got it :)

On Thu, Apr 7, 2016 at 12:29 PM, Jonathan Frederic <notifications@github.com
> wrote:

> Closed #464 <https://github.com/ipython/ipywidgets/pull/464>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipywidgets/pull/464#event-619008896>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com
